draft-packages/card/KaizenDraft/Card/Card.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react"
+const styles = require("./styles.module.scss")
+
+export interface CardProps {
+  children?: React.ReactNode
+
+  /**
+   * Not recommended. A short-circuit for overriding styles in a pinch
+   * @default ""
+   */
+  classNameAndIHaveSpokenToDST?: string
+
+  /**
+   * HTML elements that are allowed on Card.
+   */
+  tag?: "pre" | "div" | "article" | "header" | "main" | "section"
```

Comment:
These are the only tags I'm thinking we need right now. It will be easy to expand on this list later